### PR TITLE
Add downstream CSV

### PR DESCRIPTION
**Pull request by tiraboschi**
TODO: make it a template

### CODE REVIEW

**Comment by tiraboschi** [comment]
DRAFT

TODO: make it a template

**Comment by tiraboschi** [comment]
operator-courier is happy:
```
[stirabos@t470s hyperconverged-cluster-operator]$ operator-courier verify --ui_validate_io hack/downstream/
[stirabos@t470s hyperconverged-cluster-operator]$ echo $?
0
```

**Comment by djzager** [comment]
Can't we simply commit this to the downstream project instead of including this in the upstream repo?

**Comment by tiraboschi** [comment]
> Can't we simply commit this to the downstream project instead of including this in the upstream repo?

Yes, absolutely. The point is to figure out the best way to keep it aligned with:
- upstream CVS
- downstream builds

File: hack/ds_test.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Create a Catalog Source
+cat <<EOF | oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-catalogsource
+  namespace: openshift-operator-lifecycle-manager
+spec:
+  sourceType: grpc
+  image: brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/container-native-virtualization/hyperconverged-cluster-operator:v2.0.0-4
```
Comment (rthallisey) [inline hack/ds_test.sh]: this image isn't setup to use grpc to provide a CatalogSource.  We can use the hco-registry image in `deploy/converged/Dockerfile` to make the grpc bundle and point at your `downstream` CSV.  The would be hosted upstream temporarily until we sort out quay.

Comment (tiraboschi) [reply hack/ds_test.sh]: done

**Comment by mareklibra** [comment]
Please let's not forget the web-ui.

Downstream setting is done via ENV variables: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/58

**Pull request closed without merging**